socorro/processor/signature_utilities.py

```diff
@@ -662,12 +662,22 @@ def version(self):
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         return bool(raw_crash.get('AsyncShutdownTimeout'))
 
+    def get_name(self, c):
+        """Return best name for an AsyncShutdownTimeout notation condition"""
+        # The AsyncShutdownTimeout notation condition can either be a string that looks like a
+        # "name" or a dict with a "name" in it.
+        #
+        # This handles both variations.
+        if isinstance(c, dict):
+            return c.get('name')
+        return c
```

Comment:
Nit: This should probably be an inline function or a static method. Especially since the parameter is just called `c`.